scripts/shutdownLocalCluster.sh

```diff
@@ -1,21 +1,37 @@
 #!/bin/bash
+params=("$@")
 
-. `dirname $0`/cluster-run-common.sh
+lib="$(dirname $(readlink -f ${BASH_SOURCE[0]}))/cluster-run-common.sh"
+if [[ -f "$lib" ]]; then
+    . "$lib"
+else
+    echo "could not source $lib"
+    exit 1
+fi
+
+if [[ -f cluster/startup_parameters ]];then
+    if [[ -z "${params[@]}" ]]; then
+        string="$(< cluster/startup_parameters)"
+        params=( $string )
+    fi
+fi
+
+parse_args "${params[@]}"
 
 echo Number of Agents: $NRAGENTS
 echo Number of DBServers: $NRDBSERVERS
 echo Number of Coordinators: $NRCOORDINATORS
 
 if [ -z "$ONGOING_PORTS" ] ; then
-  CO_BASE=$(( $PORT_OFFSET + 8530 ))
-  DB_BASE=$(( $PORT_OFFSET + 8629 ))
-  AG_BASE=$(( $PORT_OFFSET + 4001 ))
-  SE_BASE=$(( $PORT_OFFSET + 8729 ))
+  CO_BASE=$(( PORT_OFFSET + 8530 ))
```

Comment:
Isn't the $ needed here? If not, why not?

Reply:
in math expressions the $ is not required to access variables

Reply:
OK. I hate bash.